docs/_posts/2023-03-29-functional-modularity-1.md

```diff
@@ -188,7 +188,7 @@ Thanks for taking this journey with me. Questions? Comments? I would love to con
 
 ## Footnotes
 
-[^1]: A link to a pdf for “Why Functional Programming Matters” can be found under "Background Papers" on [this page](https://www.cs.kent.ac.uk/people/staff/dat/miranda/), and a link to one of the keynote addresses mentioned can be found [here](https://www.youtube.com/watch?v=IyR04U66z7E&ab_channel=gnbitcom)
+[^1]: A link to a pdf of the original publication of “Why Functional Programming Matters” can be found under "Background Papers" on [this page](<https://www.cs.kent.ac.uk/people/staff/dat/miranda/>). However, This version contains some errors that were fixed in a later publication which can be found instead on [this page](<https://www.cse.chalmers.se/~rjmh/Papers/whyfp.html>) and a link to one of the keynote addresses mentioned can be found [here](<https://www.youtube.com/watch?v=IyR04U66z7E&ab_channel=gnbitcom>)
```

Comment:
```suggestion
[^1]: A link to a pdf of the original publication of "Why Functional Programming Matters" can be found under "Background Papers" on [this page](<https://www.cs.kent.ac.uk/people/staff/dat/miranda/>). However, This version contains some errors that were fixed in a later publication which can be found instead on [this page](<https://www.cse.chalmers.se/~rjmh/Papers/whyfp.html>) and a link to one of the keynote addresses mentioned can be found [here](https://www.youtube.com/watch?v=IyR04U66z7E&ab_channel=gnbitcom)
```

* I recommend doing a find/replace across these kinds of quotes; since they're copied from Word/Wordpress they're not internet friendly and are best replaced with standard quotes.
* Fixed the URL formatting